xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -322,9 +378,14 @@ viewers.Comments = Class.create({
    */
   doPreview : function(content, form) {
     form.previewButton._x_modePreview = true;
-    form.previewContent.update(content);
-    form.previewContent.show();
-    form.commentElt.hide();
+    const commentElt = jQuery(form).find(".wysiwyg-field");
+    if(jQuery(form).find(".commentPreview").size() === 0) {
+      commentElt.before('<div class="commentcontent commentPreview" style="display: none;"></div>');
+    }
+    const pc = jQuery(form).find(".commentcontainer .commentPreview");
```

Comment:
I have a doubt: you're looking for `.commentcontainer` or `.commentcontent` there? since you injected `commentcontent`.